packages/web/src/pages/settings-page/components/desktop/DeveloperApps/DeveloperAppListItem.tsx

```diff
@@ -56,25 +57,33 @@ export const DeveloperAppListItem = (props: DeveloperAppListItemProps) => {
 
   return (
     <li className={styles.listItem}>
-      <span className={styles.listItemIndex}>{index}</span>
+      <span className={styles.listItemImage}>
+        {imageUrl ? (
+          <img width='56' src={imageUrl} />
+        ) : (
+          <IconEmbed color='subdued' size='2xl' />
```

Comment:
nit: I'm mildly concerned that the sizes of these two things can deviate and we won't notice. Could we maybe just set the image width to 100% through a class (or use flex properties to make it grow to fit). That way the size of the content is fully controlled by the parent in one place.

Also noticing that we use a 2xl icon, which is 32px, but when it's an image it's 56px?

Reply:
Removed size from image. Have to keep on the icon because it actually renders smaller on top of a background that changes w/ darkmode